diff -ruN st-0.8.2-original/config.def.h st-0.8.2-font/config.def.h
--- st-0.8.2-original/config.def.h	2019-06-21 18:54:24.720092241 +0200
+++ st-0.8.2-font/config.def.h	2019-06-21 18:55:48.265342977 +0200
@@ -5,8 +5,8 @@
  *
  * font: see http://freedesktop.org/software/fontconfig/fontconfig-user.html
  */
-static char *font = "Liberation Mono:pixelsize=12:antialias=true:autohint=true";
-static int borderpx = 2;
+static char *font = "PragmataPro Mono:regular:pixelsize=30:antialias=true:autohint=true";
+static int borderpx = 0;

 /*
  * What program is execed by st depends of these precedence rules:
