diff -ruN code/st/st-0.8.2-original/config.def.h code/st/st-0.8.2-theme/config.def.h
--- code/st/st-0.8.2-original/config.def.h	2019-02-09 12:50:41.000000000 +0100
+++ code/st/st-0.8.2-theme/config.def.h	2019-07-05 18:14:33.234861845 +0200
@@ -82,44 +82,42 @@
  */
 unsigned int tabspaces = 8;
 
+
 /* Terminal colors (16 first used in escape sequence) */
 static const char *colorname[] = {
-	/* 8 normal colors */
-	"black",
-	"red3",
-	"green3",
-	"yellow3",
-	"blue2",
-	"magenta3",
-	"cyan3",
-	"gray90",
-
-	/* 8 bright colors */
-	"gray50",
-	"red",
-	"green",
-	"yellow",
-	"#5c5cff",
-	"magenta",
-	"cyan",
-	"white",
-
-	[255] = 0,
-
-	/* more colors can be added after 255 to use with DefaultXX */
-	"#cccccc",
-	"#555555",
+    /* 8 normal colors */
+    [0] = "#303347",           /* BLACK, HOST */
+    [1] = "#f557a0",           /* RED, SYNTAX_STRING */
+    [2] = "#a9ee55",           /* GREEN, COMMAND */
+    [3] = "#f5a255",           /* YELLOW, COMMAND_COLOR2 */
+    [4] = "#5ea2ec",           /* BLUE, PATH */
+    [5] = "#a957ec",           /* MAGENTA, SYNTAX_VAR */
+    [6] = "#5eeea0",           /* CYAN, PROMPT */
+    [7] = "#918988",           /* WHITE */
+
+    /* 8 bright colors */
+    [8] = "#918988",           /* BLACK, */
+    [9] = "#f579b2",           /* RED, COMMAND_ERROR */
+    [10] = "#bbee78",          /* GREEN, EXEC */
+    [11] = "#f5b378",          /* YELLOW, */
+    [12] = "#81b3ec",          /* BLUE, FOLDER */
+    [13] = "#bb79ec",          /* MAGENTA, */
+    [14] = "#81eeb2",          /* CYAN, */
+    [15] = "#f5eeec",          /* WHYTE, */
+
+    /* more colors can be added after 255 to use with DefaultXX */
+    [256] = "#303347", /* background */
+    [257] = "#f5eeec", /* foreground */
 };
 
-
 /*
  * Default colors (colorname index)
  * foreground, background, cursor, reverse cursor
  */
-unsigned int defaultfg = 7;
-unsigned int defaultbg = 0;
-static unsigned int defaultcs = 256;
-static unsigned int defaultrcs = 257;
+unsigned int defaultfg = 257;
+unsigned int defaultbg = 256;
+static unsigned int defaultcs = 257;
+static unsigned int defaultrcs = 256;
 
 /*
  * Default shape of cursor
